src/instance/DataModelInstance.ts

```diff
@@ -46,19 +57,22 @@ export default class DataModelInstance {
       this.root.set(rootName, new ContainerInstance(modelRoot, instance.get('./*[1]')!, null));
     } else {
       this.root.set(rootName, new ContainerInstance(modelRoot, Object.values(instance)[0], null));
-      this.rawInstance = this.toXML().root()!;
-
-      this.visit(instanceToVisit => {
-        if (
-          instanceToVisit instanceof ListChildInstance ||
-          instanceToVisit instanceof LeafListChildInstance ||
-          instanceToVisit instanceof LeafInstance ||
-          instanceToVisit instanceof ContainerInstance
-        ) {
-          const xPath = getPathXPath(instanceToVisit.getPath());
-          instanceToVisit.setConfig(this.rawInstance.get(xPath, this.model.namespaces)!);
-        }
-      });
+
+      if (!options?.jsonMode) {
```

Comment:
keeping XML around eh? Unfortunately that means we'll have to continue to pull that dumb libxmljs